python_modules/dagster/dagster/_core/instance/config.py

```diff
@@ -430,6 +430,11 @@ def dagster_instance_config_schema() -> Mapping[str, Field]:
                         "How many threads to use to process ticks from multiple automation policy sensors in parallel"
                     ),
                 ),
+                "run_submission_batch_size": Field(
```

Comment:
i don't have a better name here but at first glance i would assume that this means that the runs are submitted in batches in the queue, not that they are submitted one by one or in a threadpool and then checked for cancellation after this many runs

i guess something like 'run_cancellation_checkpoint_size' or something? Don't love that either

Reply:
yeah good point. 

`tick_cancellation_check_interval`
`sensor_cancellation_check_interval`
cancellation, termination...